packages/react/src/components/Text/Text.tsx

```diff
@@ -103,8 +103,6 @@ class Text extends UIComponent<WithAsProp<TextProps>, any> {
 Text.create = createShorthandFactory({ Component: Text, mappedProp: 'content' })
 
 /**
- * A Text component formats occurrences of text consistently.
- * @accessibility
- * Text is how people read the content on your website.
+ * A Text component is used to consistently style and format occurrences of text.
```

Comment:
We didn't have any description with `is used` so far in the Components. Can we reformulate this description in a similar format as the others?

Reply:
have replaced `used to` with active-voice verbs